tests/integration/targets/azure_rm_recoveryservicesvault/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+- name: Fix resource prefix
+  set_fact:
+    resource_group: 'MyResourceGroup'
+    name: 'MyRecoveryVault'
```

Comment:
Please delete these lines (3,6,7).  And change the name and location to the following, the purpose is to cooperate with pipeline automated testing!
```e.g
         name: "revault{{ resource_group | hash('md5') | truncate(22, True, '') }}"
         location: "eastus"

Reply:
The file is adjusted accordingly.